python/lsst/pipe/tasks/multiBand.py

```diff
@@ -116,14 +116,19 @@ class DetectCoaddSourcesConfig(PipelineTaskConfig, pipelineConnections=DetectCoa
     detection = ConfigurableField(target=DynamicDetectionTask, doc="Source detection")
     coaddName = Field(dtype=str, default="deep", doc="Name of coadd")
     doInsertFakes = Field(dtype=bool, default=False,
-                          doc="Run fake sources injection task")
+                          doc="Run fake sources injection task",
+                          deprecated=("doInsertFakes is no longer supported. This config will be removed "
+                                      "after v24."))
     insertFakes = ConfigurableField(target=BaseFakeSourcesTask,
                                     doc="Injection of fake sources for testing "
-                                    "purposes (must be retargeted)")
+                                    "purposes (must be retargeted)",
+                                    deprecated=("insertFakes is no longer supported. This config will "
+                                                "be removed after v24."))
     hasFakes = Field(
         dtype=bool,
         default=False,
-        doc="Should be set to True if fake sources have been inserted into the input data."
+        doc="Should be set to True if fake sources have been inserted into the input data.",
+        deprecated="hasFakes is no longer supported. This config will be removed after v24."
     )
```

Comment:
This option is from the newer round of fakes insertion changes and is probably going to be deprecated as part of the moving to gen 3 only but might be a different round of removals.

Reply:
It needs to have the same deprecation warning as processCcdWithFakes and friends.